tests/msc2716_test.go

```diff
@@ -1211,3 +1254,159 @@ func batchSendHistoricalMessages(
 
 	return res
 }
+
+// Verify that the batch of historical messages looks correct and in order in
+// the message scrollback. We can also optionally check that the historical
```

Comment:
"in order in the message scrollback" - please specify if it's old -> new or new -> old.

Reply:
Also, I would actually write `/messages` rather than "message scrollback". The former is much clearer that this function is going to be doing an HTTP hit to verify the events.